api/src/pcapi/routes/backoffice/templates/offer/details/stocks.html

```diff
@@ -0,0 +1,22 @@
+<turbo-frame id="offer_stocks_frame">
+  <table class="table mb-4 my-4">
+    <thead>
+      <tr>
+        <th scope="col-1">ID</th>
```

Comment:
Pour que ce soit plus clair pour l'utilisateur : 
```suggestion
        <th scope="col-1">Stock ID</th>
```